Bottom: 3ebbda102be755a7cb898bfc5a7cecfd0dca91eb
Top:    51cf32067f75f468cd29adb02276e26cac47e4cf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-15 18:57:32 +0100

Add some missing files to POTFILES.in


---

diff --git a/po/POTFILES.in b/po/POTFILES.in
index c3eb4ea..d3d3bbb 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -21,11 +21,15 @@ src/hid/batch/batch.c
 src/hid/common/actions.c
 src/hid/common/hidnogui.c
 src/hid/gerber/gerber.c
+src/hid/gtk/ghid-cell-renderer-visibility.c
+src/hid/gtk/ghid-coord-entry.c
+src/hid/gtk/ghid-main-menu.c
+src/hid/gtk/ghid-route-style-selector.c
 src/hid/gtk/gtkhid-main.c
 src/hid/gtk/gui-command-window.c
 src/hid/gtk/gui-config.c
-src/hid/gtk/gui-dialog-print.c
 src/hid/gtk/gui-dialog.c
+src/hid/gtk/gui-dialog-print.c
 src/hid/gtk/gui-drc-window.c
 src/hid/gtk/gui-keyref-window.c
 src/hid/gtk/gui-library-window.c
@@ -33,6 +37,7 @@ src/hid/gtk/gui-log-window.c
 src/hid/gtk/gui-misc.c
 src/hid/gtk/gui-netlist-window.c
 src/hid/gtk/gui-top-window.c
+src/hid/lesstif/dialogs.c
 src/hid/ps/ps.c
 src/main.c
 src/misc.c
